### ISSUE TITLE
Request: parse multiple sections with the same name

### ISSUE THREAD

**Issue by akdor1154** [cemdervis/SharpConfig#25]
Hi,
It would be awesome if SharpConfig could actually cope with multiple sections with the same name, instead of ignoring them. Even if `config["sectionName"]` just returned the first one, it would be very useful if `foreach (Section section in config)`  would properly enumerate all of them.
Example configuration:
```
[server]
ip = 192.168.0.1

[client]
ip = 192.168.0.8
os = Windows

[client]
ip = 192.168.0.9
os = Linux
```

**Comment by akdor1154** [comment]
Looking through the code, it looks like this would actually be very, very simple to add, with the caveat that Getting and Deleting by section name would get or delete the first matching section if there were multiple. For my purposes, this would be absolutely fine, as there are ways to get or delete sections by Section object or by index anyway. Would you accept a pull request enabling this?

**Comment by cemdervis** [comment]
This is actually a good idea, but as you said, would go against the existing model of getting and setting sections/settings. This is not a problem however, just make sure you add a (static) flag to the Configuration class that controls this behaviour. I would gladly accept such a pull request, thank you!

**Comment by cemdervis** [comment]
This has been fixed with the new version (1.5.5).
